config/daemon/systemd.md

```diff
@@ -34,14 +34,14 @@ do not have `systemctl`, use the `service` command.
 ### Start automatically at system boot
 
 If you want Docker to start at boot, see
-[Configure Docker to start on boot](/engine/install/linux-postinstall.md#configure-docker-to-start-on-boot).
+[Configure Docker to start on boot](../../engine/install/linux-postinstall.md#configure-docker-to-start-on-boot).
 
 ## Custom Docker daemon options
 
 There are a number of ways to configure the daemon flags and environment variables
 for your Docker daemon. The recommended way is to use the platform-independent
 `daemon.json` file, which is located in `/etc/docker/` on Linux by default. See
-[Daemon configuration file](/engine/reference/commandline/dockerd.md#daemon-configuration-file).
+[Daemon configuration file](../../engine/reference/commandline/dockerd.md#daemon-configuration-file).
```

Comment:
Link doesn't work. Could it be due to the fact that the topic is autogenerated?

Reply:
Ah, yes, it's because Netlify doesn't fetch the remove sources; the link looks correct; https://deploy-preview-10615--docsdocker.netlify.com/engine/reference/commandline/dockerd/#daemon-configuration-file

And it does work locally